src/main/java/com/marklogic/appdeployer/command/security/DeployRolesCommand.java

```diff
@@ -46,6 +46,7 @@ public void execute(CommandContext context) {
 
 	@Override
 	protected String adjustPayloadBeforeSavingResource(ResourceManager mgr, CommandContext context, File f, String payload) {
+		payload = adjustPayloadBeforeSavingResource(mgr, context, f, payload);
```

Comment:
I think this needs to be payload = super.adjustPayload, otherwise it will be a stackoverflow error.